package.json

```diff
@@ -78,6 +80,7 @@
     "setupTestFrameworkScriptFile": "./test-setup.js"
   },
   "dependencies": {
+    "babel-runtime": "^6.26.0",
```

Comment:
You didn't mean to add this dependency, did you?

Reply:
It's not required for Chromatic.

Reply:
Nope, good catch, that was part of the old branch.